commands/rm.go

```diff
@@ -149,13 +149,23 @@ func rmAllInactive(ctx context.Context, txn *store.Txn, dockerCli command.Cli, i
 		builders[i] = &nginfo{ng: ng}
 	}
 
+	ctxbuilders, err := dockerCli.ContextStore().List()
+	if err != nil {
+		return err
+	}
+
 	eg, _ := errgroup.WithContext(ctx)
 	for _, b := range builders {
 		func(b *nginfo) {
 			eg.Go(func() error {
 				if err := loadNodeGroupData(ctx, dockerCli, b); err != nil {
 					return errors.Wrapf(err, "cannot load %s", b.ng.Name)
 				}
+				for _, cb := range ctxbuilders {
+					if b.ng.Driver == "docker" && len(b.ng.Nodes) == 1 && b.ng.Nodes[0].Endpoint == cb.Name {
+						return errors.Errorf("context builder cannot be removed, run `docker context rm %s` to remove this context", cb.Name)
```

Comment:
I wonder if we should just `return` to skip over the context builders here, instead of returning an error.

Reply:
I see what you mean but I think the user would expect this builder to be removed and if we don't return an error he will be confused imo.